pkg/acceptance/helpers/external_volume_client.go

```diff
@@ -21,39 +20,56 @@ func NewExternalVolumeClient(context *TestClientContext, idsGenerator *IdsGenera
 	}
 }
 
-func (c *ExternalVolumeClient) exec(sql string) error {
-	ctx := context.Background()
-	_, err := c.context.client.ExecForTests(ctx, sql)
-	return err
+func (c *ExternalVolumeClient) client() sdk.ExternalVolumes {
+	return c.context.client.ExternalVolumes
 }
 
-// TODO(SNOW-999142): Use SDK implementation for External Volume once it's available
+// TODO switch to returning *sdk.ExternalVolume
+// need to update existing acceptance tests for this
 func (c *ExternalVolumeClient) Create(t *testing.T) (sdk.AccountObjectIdentifier, func()) {
 	t.Helper()
+	ctx := context.Background()
+
 	id := c.ids.RandomAccountObjectIdentifier()
-	err := c.exec(fmt.Sprintf(`
-create external volume %s
-	storage_locations =
-    	(
-    		(
-            	name = 'my-s3-us-west-2'
-            	storage_provider = 's3'
-            	storage_base_url = 's3://my_example_bucket/'
-            	storage_aws_role_arn = 'arn:aws:iam::123456789012:role/myrole'
-            	encryption=(type='aws_sse_kms' kms_key_id='1234abcd-12ab-34cd-56ef-1234567890ab')
-        	)
-      	);
-`, id.FullyQualifiedName()))
+	kmsKeyId := "1234abcd-12ab-34cd-56ef-1234567890ab"
+	storageLocations := []sdk.ExternalVolumeStorageLocation{
+		{
+			S3StorageLocationParams: &sdk.S3StorageLocationParams{
+				Name:              "my-s3-us-west-2",
+				StorageProvider:   "S3",
+				StorageAwsRoleArn: "arn:aws:iam::123456789012:role/myrole",
+				StorageBaseUrl:    "s3://my_example_bucket/",
+				Encryption: &sdk.ExternalVolumeS3Encryption{
+					Type:     "AWS_SSE_KMS",
+					KmsKeyId: &kmsKeyId,
+				},
+			},
+		},
+	}
+
+	req := sdk.NewCreateExternalVolumeRequest(id, storageLocations)
+	err := c.client().Create(ctx, req)
 	require.NoError(t, err)
 
+	_, showErr := c.client().ShowByID(ctx, id)
+	require.NoError(t, showErr)
+
 	return id, c.DropFunc(t, id)
```

Comment:
Initially I wanted to avoid updating other acceptance test files using this function (e.g. database_acceptance_test.go, secondary_database_acceptance_test.go, grant_privileges_to_account_role_acceptance_test.go) in this MR as it's already quite big, and I've had some issues running other acceptance tests with my local setup. Happy to update the tests now though if you like, may need you to run them to check they're still green if I do.